docs/input-jdbc.asciidoc

```diff
@@ -373,6 +374,30 @@ result-set. The limit size is set with `jdbc_page_size`.
 
 Be aware that ordering is not guaranteed between queries.
 
+[id="plugins-{type}s-{plugin}-jdbc_paging_avoid_count"]
+===== `jdbc_paging_avoid_count`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is `false`
+
+Whether to use count query during the JDBC paging
+
+If false, a count query and multiple paged queries (with `LIMIT` statement, etc.) are used under the hood.
+
+Otherwise, if true, multiple paged queries (without a count query ahead) will be performed until no more rows are retrieved.
```

Comment:
Maybe is better the  "if true" so it's symmetric to the `false` case, a couple of eyes from @karenzone  on the documentation part are far better then mine on this.

```suggestion
If true, multiple paged queries (without a count query ahead) will be performed until no more rows are retrieved.
```